src/dispatchers.ts

```diff
@@ -132,33 +130,7 @@ export function createFollowHandler(accountService: AccountService) {
             return;
         }
 
-        const currentFollowers =
-            (await ctx.data.db.get<string[]>(['followers'])) ?? [];
-        const shouldRecordFollower =
-            currentFollowers.includes(sender.id.href) === false;
-
-        // Add follow activity to inbox
-        const followJson = await follow.toJsonLd();
-
-        ctx.data.globaldb.set([follow.id.href], followJson);
-        await addToList(ctx.data.db, ['inbox'], follow.id.href);
```

Comment:
We want to keep the `inbox` and the follow storage for now because we haven't migrated away from the key_value for these - only for the followings and followers!